src/nvim/api/window.c

```diff
@@ -432,3 +432,41 @@ Boolean nvim_win_is_valid(Window window)
   return ret;
 }
 
+
+/// Configure window position. Currently this is only used to configure
+/// floating and external windows (including changing a split window to these
+/// types).
+///
+/// See documentation at |nvim_open_window|, for the meaning of parameters.
```

Comment:
tag `|nvim_open_window|` won't exist, should be `|nvim_open_window()|`

Reply:
```suggestion
/// See documentation at |nvim_open_win()|, for the meaning of parameters.
```

s/window/win/